setup.py

```diff
@@ -53,7 +53,7 @@
     "docs": [
         "Sphinx~=3.0",
         "myst_parser>=0.13",
-        "sphinx_book_theme==0.38.0",
+        "sphinx_book_theme==0.0.38",
```

Comment:
```suggestion
        "sphinx-book-theme>=0.0.33",
```
This would match docs/requirements.txt. (We should probably pin there and keep it looser here eventually)